src/NewTwitchApi/Resources/StreamsApi.php

```diff
@@ -12,17 +12,17 @@ class StreamsApi extends AbstractResource
     /**
      * @throws GuzzleException
      */
-    public function getStreamForUserId(string $userId): ResponseInterface
+    public function getStreamForUserId(string $userId, string $bearer = null): ResponseInterface
```

Comment:
Just a thought for discussion, but please do not change your code as it is the safest, BC change. I wonder, if bearer is truly required, if perhaps this library should eventually enforce it. Thoughts?

Reply:
Requests will already be broken for those who are not sending a bearer, so I personally see no issue with requiring it in the library, it would likely save some people thinking they don't need to send the bearer when in fact they do.